published/035-2019-mar-26.md

```diff
@@ -0,0 +1,72 @@
+
+
+## Meet Jia Le Dai
+
+[Jiale Dai](https://twitter.com/daijiale6239) used to work on mapping applications as a backend engineer at Baidu. In 2017 he became interested in IPFS, libp2p, and multiformats. Jiale eventually started organizing [IPFS meetups for Chinese developers and enthusiasts](https://mp.weixin.qq.com/s/NrvZo5_hLD_CIXpZFPARbg). His goal is to get more Chinese developers interested in distributed web technologies like IPFS. Jiale joins the IPFS Weekly Call next Monday at 5pm UTC to present on IPFS in China.
+
+
+## IPFS in the wild
+*Do you follow [IPFS on Twitter](https://twitter.com/IPFSbot)? For the latest mentions of IPFS in the news, check our Twitter feed or see the [latest articles on Awesome IPFS](https://awesome.ipfs.io/categories/articles/).* 
+
++ [Sunsetting Mist: What’s ahead for our team](https://medium.com/@avsa/sunsetting-mist-da21c8e943d2), 22 Mar 2019
++ [IPFS Encryption](https://medium.com/@theotherfruit/ipfs-encryption-492b004659d1), 21 Mar 2019
++ [Numerai raises $11M with ICO & prepares to launch prediction marketplace](https://ayo.news/2019/03/22/numerai-raises-11m-with-ico-prepares-to-launch-prediction-marketplace/), 22 Mar 2019
++ [Hugo and IPFS: how this blog works (and scales to serve 5,000% spikes instantly!)](https://withblue.ink/2019/03/20/hugo-and-ipfs-how-this-blog-works-and-scales.html), 20 Mar 2019
++ [Textile Notes: A minimalist tool for your creative ideas](https://medium.com/textileio/textile-notes-a-minimalist-tool-for-your-creative-ideas-68b9357d5cd0), 19 Mar 2019
++ PODCAST: [WR 79: Another internet is possible: IPFS](https://www.danielprimo.io/blog/otra-internet-es-posible-ipfs), Web Reactiva, 19 Mar 2019
++ [Society Desperately Needs An Alternative Web](https://www.forbes.com/sites/cognitiveworld/2019/03/15/society-desperately-needs-an-alternative-web/#155cd17424e3), Forbes, 15 Mar 2019
++ [A picture is worth a thousand crypto](https://thecreativecrypto.com/creating-a-creative-economy-interview-with-the-founders-of-crea/), The Creative Crypto, 1 Jan 2019
+
+
+## Updates and new releases
+*See the latest releases of IPFS tools and projects across the ecosystem.*
+
++ [Pony 0.28.0 has been released.](https://www.ponylang.io/blog/2019/03/0.28.0-released/)
++ [DTube IPFS Discord Bot March 2019 Update!](https://steemit.com/ipfs/@techcoderx/dtube-ipfs-discord-bot-march-2019-update-improved-file-pinning-node-stats-usage-data-command-and-more)
++ [Radium 2.0 Development & Expansion Roadmap](https://blog.radiumcore.org/radium-2-0-development-expansion-roadmap-updated-march-2019-382041249cf7) 
+
+
+## Tools and projects we <3
+*[Awesome IPFS](https://awesome.ipfs.io/) is a community maintained and updated list of projects, tools, or pretty much any things related to IPFS that are totally awesome. To see more, or add yours to the list, visit [Awesome IPFS on GitHub](https://github.com/ipfs/awesome-ipfs).* 
+
++ [Qri was on the front page of Hacker News](https://news.ycombinator.com/item?id=19470064)
```

Comment:
Ditto as previous comment, eg "Qri, a community effort to feed leftover cake to neighborhood squirrels, was on the front page of HN!"